package.json

```diff
@@ -15,9 +18,8 @@
     "dropbox": "10.31.0",
     "immutable": "^3.8.2",
     "lodash": "^4.17.21",
-    "node-sass": "4.13",
     "pegjs": "^0.10.0",
-    "react": "^16.8.3",
+    "react": "16.10.0",
```

Comment:
Just a minimal update to get rid of a warning. Upgrading further is absolutely reasonable, of course!